data/strings/strings.txt

```diff
@@ -20715,7 +20717,7 @@
 
   [enable_logging_warning_message]
     tags = android
-    en = The option turns on logging for diagnostic purposes. It can be helpful for our team to troubleshoot issues with the app. Enable this option temporarily to record and send detailed logs about your issue to us.
+    en = Enable this option temporarily to record and send detailed diagnostic logs about your issue to us.
```

Comment:
This should caution that it may send location or destination info.

Reply:
done